README.md

```diff
@@ -15,3 +15,28 @@ Visit [https://assembly.com](https://assembly.com) to learn more.
     git clone git@github.com:asm-products/signupsumo-web.git
     cd signupsumo-web
     bin/setup
+
+### Run using Docker
+
+- [Install Boot2Docker](http://boot2docker.io/)
+- Run Boot2Docker
+  - $ `boot2docker init`
+  - $ `boot2docker start`
+  - $ `$(boot2docker shellinit)`
+  - To persist the environment variables across shell sessions, you can add `$(boot2docker shellinit)` to your `~/.bashrc` file
+- [Install Fig](http://www.fig.sh/install.html)
+  - `curl -L https://github.com/docker/fig/releases/download/1.0.0/fig-`uname -s`-`uname -m` > /usr/local/bin/fig; chmod +x /usr/local/bin/fig`
+- Build the Docker containers
+  - $ `fig build`
+- Run the app
+  - $ `fig up`
+- Migrate the database
+  - Do this in another terminal window
+    - `fig run web rake db:create`
+- Access the app locally
+  - Run `boot2docker ip` to get the ip address of the Docker daemon
+  - View the app at `IP_ADDRESS:3000`
```

Comment:
Yeah, might be nice, though a fair amount of people should already have boot2docker installed and maybe even fig.